test/jasmine/tests/table_test.js

```diff
@@ -0,0 +1,135 @@
+var Plotly = require('@lib');
+var Lib = require('@src/lib');
+var d3 = require('d3');
+var createGraphDiv = require('../assets/create_graph_div');
+var destroyGraphDiv = require('../assets/destroy_graph_div');
+
+describe('table plots', function() {
+    'use strict';
+    var gd;
+
+    beforeEach(function() {
+        gd = createGraphDiv();
+    });
+
+    afterEach(destroyGraphDiv);
+
+    function initialize(mock, gd) {
+        return Plotly.plot(gd, mock.data, mock.layout);
+    }
+
+    function countCells() {
+        return d3.selectAll('.cell-rect').size();
+    }
+
+    it('should be able switch to table trace from another plot type', function(done) {
+        var mock = Lib.extendDeep({}, require('@mocks/workspace_starter_state.json'));
```

Comment:
@VeraZab would you mind replacing this `require` call with:

```js
var mock = {
  data: //
  layout //
}
```

as each json mock in `test/image/mocks/` is expected to have a corresponding image in `test/image/baselines/`? This should make the test go :white_check_mark:

Reply:
sure, thanks!